python/tests/test_3_offline.py

```diff
@@ -257,7 +257,7 @@ def test_chat_idempotent(self, chat1, ac1):
         assert chat2.id == chat1.id
         assert chat2.get_name() == chat1.get_name()
         assert chat1 == chat2
-        assert not (chat1 != chat2)
+        assert chat1 == chat2
```

Comment:
probably the intention here was to test the `__ne__` implementation, so converting to `==`/`__eq__` defeats the purpose